include/seqan/graph_types/graph_impl_tree.h

```diff
@@ -639,6 +639,57 @@ getAdjacencyMatrix(Graph<Tree<TCargo, TSpec> > const& g,
 
 //////////////////////////////////////////////////////////////////////////////
 
+template<typename TVector, typename TCargo, typename TSpec, typename TVertex>
+inline void
+getVertexAdjacencyVector(TVector & vectIn,
+                         TVector & vectOut,
+                         Graph<Tree<TCargo, TSpec> > const & g,
+                         TVertex const & vertex)
+{
```

Comment:
is this function the same as in graph_directed? If yes, please make a single function, as a generic default, and only overload for those cases that need a special case.